### DISCUSSION TITLE
[CRUD writable views] When calling PATCH endpoint to an exposed view it rewrites the path to /public

### DISCUSSION THREAD

**Discussion by alexBonadei** [mia-platform/community#329] [Q&A]
### Which section?

Marketplace

### Which is the version?

.

### Description

Hi.

I've set a view called `papers_view` and exposed it to the `/papers-view` endpoint. I also set the `PATCH /:id` route since it's a writable view.

When I call this endpoint like this: `/papers-view/6526539f30e2e9c5a199a482` , the crud-service rewrite the path this way: `/public/papers-view/6526539f30e2e9c5a199a482`. The crud-service then replies with a 404.

I checked everywhere but I don't know where is set this rewrite.

Am I missing something? Thank you in advance.

**Comment by fredmaggiowski** [comment]
Hi, are you contacting the CRUD through an endpoint exposed by the API Gateway?

**Reply by alexBonadei** [reply]
Yes a setup the endpoint in the dedicated section, selecting MongoDB View as the type
